airflow/jobs/scheduler_job_runner.py

```diff
@@ -2066,8 +2066,13 @@ def _orphan_unreferenced_assets(self, session: Session = NEW_SESSION) -> None:
             )
         )
 
-        updated_count = sum(self._set_orphaned(asset) for asset in orphaned_asset_query)
-        Stats.gauge("asset.orphaned", updated_count)
+        orphaning_identifiers = list(self._get_orphaning_identifier(asset) for asset in orphaned_asset_query)
```

Comment:
why do we want to use `list` instead of `[]` here?

Reply:
I think I simply forgot to change it’s a thing (it’s easier to change `sum` to `list`…)

I’ll use a list comprehension instead.